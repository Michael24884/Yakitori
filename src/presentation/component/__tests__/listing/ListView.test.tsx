/**
 * @format
 */
import React from 'react';
import {ListView} from '@components';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const instance = renderer.create(<ListView />);
  expect(instance.toJSON()).toMatchSnapshot();
});
