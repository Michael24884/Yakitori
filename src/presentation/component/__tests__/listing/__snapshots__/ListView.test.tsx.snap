// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
    ]
  }
>
  <RCTScrollView
    ListEmptyComponent={<EmptyListView />}
    collapsable={false}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    refreshControl={
      <RefreshControlMock
        refreshing={false}
      />
    }
    removeClippedSubviews={false}
    renderItem={[Function]}
    renderScrollComponent={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "flex": 1,
      }
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <RCTRefreshControl />
    <View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
          text="Oops!"
        >
          Oops!
        </Text>
        <Text
          style={
            Object {
              "color": "#4f4f4f",
              "fontSize": 8,
            }
          }
          text="Không có dữ liệu!"
        >
          Không có dữ liệu!
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
